cmd/lakectl/cmd/doctor.go

```diff
@@ -11,11 +11,42 @@ import (
 	"github.com/treeverse/lakefs/pkg/api"
 )
 
-var (
-	ErrCredential       = errors.New("credential error")
-	ErrWrongEndpointURI = errors.New("wrong endpoint_uri error")
-	ErrUnknownConfig    = errors.New("unknown configuratioin error")
-)
+type ErrCredential struct {
```

Comment:
Error _types_ look like `CredentialsError`; error _base variables_ look like `ErrCredentials`.  Yes, Go errors have many layers and their implementation reflects their history.
Sorry.